backend/app/exceptions/model_exceptions.py

```
@@ -0,0 +1,13 @@
+class HuggingFaceAPIError(Exception):
+    """Exception raised for errors related to the HuggingFace API."""
+    def __init__(self, status_code: int, response_text: str):
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)